docs/tutorials/PickupGuide.md

```diff
@@ -0,0 +1,77 @@
+PickupGuide
+===========
+
+### From open.mp Wiki
+
+  
+
+
+### Define the pickupid
+
+The first thing to be done when creating pickups is creating a place to store their ID. This will be done in a global variable so it can be set when you create the pickup and read when you pick up a pickup, calling a callback with the ID of the pickup you picked up. For this example we will use the name "mypickup".
```

Comment:
I think putting it this way is better:

_The first thing to do when creating a pickup is to store it's identifier (ID) in a variable.  This maye be done by creating a global variable and then assigning the pickup identifier (ID) to it, so it can be used to detect which pickup was picked up by the player and more.  For this example, we'll use the name "MyPickup"._

- This way it follows the naming convention of variables according to the Pawn Curriculum by Y_Less.
- More explicit in what we need to do and for what reason.

Reply:
> For this example, we'll use the name "MyPickup".
> This way it follows the naming convention of variables according to the Pawn Curriculum by Y_Less.

Not quite. Variables should be named in `camelCase`, not `PascalCase`. See the [Style Guide](https://wiki.open.mp/docs/scripting/language/Style#local-variables)